test/states/test_projections_benchmark.py

```diff
@@ -75,9 +83,11 @@ def test_benchmark_nbody_projection(
 ):
     """Test the mean field projection over different states,
     and using both implementations"""
-    op_test = TEST_OPERATORS[op_name]
     print("testing the consistency of projection in", op_name)
-    op_sq = op_test * op_test
+    op_sq = TEST_OPERATORS_SQ[op_name]
+
+    if "tom_projection" == projection_name and state_name == "None":
```

Comment:
"tom_projection" does not support `None` as an alias of the maximally mixed state, so we can not process the corresponding tests.